src/bioregistry/analysis/paper_ranking.py

```diff
@@ -63,9 +65,40 @@ def load_bioregistry_json(file_path):
     return pd.DataFrame(publications)
 
 
-def fetch_pubmed_papers():
-    """Fetch PubMed papers from the last 30 days using specific search terms.
+def load_curated_papers(file_path=CURATED_PAPERS_PATH):
+    """Load curated papers data from TSV file, and fetch titles and abstracts for PMIDs.
+
+    :param file_path: Path to the curated_papers.tsv file.
+    :type file_path: str
+    :return: DataFrame containing curated publication details.
+    :rtype: pd.DataFrame
+    """
+    curated_df = pd.read_csv(file_path, sep="\t")
+    curated_df = curated_df.rename(columns={"pmid": "pubmed", "relevant": "label"})
+    curated_df["title"] = ""
+    curated_df["abstract"] = ""
+
+    pmids_to_fetch = curated_df["pubmed"].tolist()
+    fetched_metadata = {}
+    for chunk in [pmids_to_fetch[i : i + 200] for i in range(0, len(pmids_to_fetch), 200)]:
```

Comment:
there's an (more?)itertools function for chunking that will make this code more idiomatic